deltachat-ios/Controller/ChatViewController.swift

```diff
@@ -274,12 +274,12 @@ class ChatViewController: MessagesViewController {
 
         // Set outgoing avatar to overlap with the message bubble
         layout?.setMessageIncomingMessageTopLabelAlignment(LabelAlignment(textAlignment: .left,
-            textInsets: UIEdgeInsets(top: 0, left: 18, bottom: outgoingAvatarOverlap, right: 0)))
+            textInsets: UIEdgeInsets(top: 0, left: 18, bottom: 0, right: 0)))
         layout?.setMessageIncomingAvatarSize(CGSize(width: 30, height: 30))
         layout?.setMessageIncomingMessagePadding(UIEdgeInsets(
-            top: -outgoingAvatarOverlap, left: -18, bottom: outgoingAvatarOverlap / 2, right: 18))
+            top: 0, left: -18, bottom: 0, right: 0))
         layout?.setMessageIncomingMessageBottomLabelAlignment(LabelAlignment(textAlignment: .left,
-            textInsets: UIEdgeInsets(top: -7, left: 38, bottom: 0, right: 0)))
+            textInsets: UIEdgeInsets(top: 0, left: 12, bottom: 0, right: 0)))
```

Comment:
what are these changes for?
the removed outgoingAvatarOverlap and the changed UIEdgeInsets?

Reply:
There was no need to have that bottom padding anymore. 
All labels have been shifted so that the avatar overlaps them a little bit less than it overlaps now. Removing some of these magic numbers reduces the complexity to understand how the message view is layouted. Also the cell heights of the left sided views can be same as the ones on the right side (see changes on l. 757 and l. 769), which again takes a way a bit of complexity.

Reply:
great!